src/Compilers/Test/Core/Assert/ConditionalFactAttribute.cs

```diff
@@ -159,6 +159,7 @@ public static class ExecutionConditionUtil
         public static bool IsWindowsDesktop => IsWindows && IsDesktop;
         public static bool IsMonoDesktop => Type.GetType("Mono.Runtime") != null;
         public static bool IsMono => MonoHelpers.IsRunningOnMono();
+        public static bool IsMonoCore => Type.GetType("Mono.RuntimeStructs") != null;
```

Comment:
Consider adding comments saying `IsMonoCore` means https://github.com/dotnet/runtime/tree/main/src/mono and `IsMonoDesktop` means https://github.com/mono/mono or something like that.